### PR TITLE
feat: init no op mtls policy

### PR DESCRIPTION
**Pull request by ytvnr**
https://gravitee.atlassian.net/browse/APIM-3980

## Description

A small description of what you did in that PR.

## Additional context

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.0.0`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-mtls/1.0.0/gravitee-policy-mtls-1.0.0.zip)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+    <properties>
+        <gravitee-bom.version>8.1.0</gravitee-bom.version>
+        <gravitee-gateway-api.version>3.5.1</gravitee-gateway-api.version>
+        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
+        <gravitee-apim.version>4.4.3</gravitee-apim.version>
+
+        <maven-plugin-assembly.version>3.5.0</maven-plugin-assembly.version>
+        <maven-plugin-prettier.version>0.19</maven-plugin-prettier.version>
+        <maven-plugin-prettier.prettierjava.version>2.1.0</maven-plugin-prettier.prettierjava.version>
```
Comment (phiz71) [inline pom.xml]: I think these version are handled by the parent. You don't need to specify them anymore.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                    <prettierJavaVersion>${maven-plugin-prettier.prettierjava.version}</prettierJavaVersion>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
```
Comment (phiz71) [inline pom.xml]: I think it's already defined in the gravitee-parent pom. So probably it could be removed here.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>22.0.2</version>
+    </parent>
+
+    <properties>
+        <gravitee-bom.version>8.1.0</gravitee-bom.version>
+        <gravitee-gateway-api.version>3.5.1</gravitee-gateway-api.version>
+        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
+        <gravitee-apim.version>4.4.3</gravitee-apim.version>
```
Comment (phiz71) [inline pom.xml]: Does it mean we don't want to support APIM version lower to 4.4.x ?

**Review by phiz71 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (ytvnr) [reply pom.xml]: thanks :)

File: pom.xml (hunk 3 above)
Comment (ytvnr) [reply pom.xml]: it will be for the next 4.5, so it is reasonable to test with last APIM release, wdyt ?

**Review by phiz71 — APPROVED** [review]

**Pull request merged**